packages/Core/src/panda/recipes/button.ts

```diff
@@ -0,0 +1,420 @@
+import { defineRecipe } from '@pandacss/dev'
+
+export const button = defineRecipe({
+  className: 'wui-button',
+  description: 'welcome-ui button',
+  jsx: [/Button.*/],
+  base: {
+    color: 'light-900',
+    fontWeight: 'bold',
+    letterSpacing: 0,
+    position: 'relative',
+    display: 'inline-flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    width: 'auto',
+    textDecoration: 'none',
+    textAlign: 'center',
+    whiteSpace: 'nowrap',
+    cursor: 'pointer',
+    outline: 'none !important' /* important for firefox */,
+    border: 'sm',
+    borderRadius: 'md',
+    appearance: 'none',
+    overflow: 'hidden',
+    transition: 'medium',
```

Comment:
Do we want to have them in a particular order (alphabetical, or by "groups" (position, box model, etc)) or yolo is fine?

Reply:
I just c/c the style inside from `packages/Button/styles.ts`

Reply:
I think so, but maybe later :grin: